db/migrate/20200225111846_create_spaces.rb

```diff
@@ -6,7 +6,7 @@ def change
       t.integer :price
       t.datetime :date_start
       t.datetime :date_end
-      t.references :users, foreign_key: true
+      t.references :user, foreign_key: true
```

Comment:
Good to check these changes don't affect how the databases function

Reply:
it works - We had a lot of dependencies on singular implementation. Personally I think singular makes more sense